scripts/code_analyze.sh

```diff
@@ -3,5 +3,6 @@ flutter pub get && flutter pub run build_runner build --delete-conflicting-outpu
 flutter pub run import_sorter:main --no-comments
 flutter format lib/ test/
 flutter pub get
-flutter analyze --no-fatal-infos --no-fatal-warnings
+flutter analyze
```

Comment:
IMO, `flutter analyze --no-fatal-infos --no-fatal-warnings` should be left as is. 
Treating `infos` or `warnings` as `fatal` is too strict at a time when the project is still in its early stages.